.kitchen.yml

```diff
@@ -1,78 +1,37 @@
 ---
 driver:
-  name: dokken
+  name: vagrant
```

Comment:
Yeah so faster tests + CI but no windows development support.I would understand if the former was deemed more valuable.